recipes/mvapich2/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = "3.0" %}
+
+package:
+  name: mvapich
+  version: {{ version }}
+
+source:
+  url: https://mvapich.cse.ohio-state.edu/download/mvapich/mv2/mvapich-{{ version }}.tar.gz
+  sha256: ee076c4e672d18d6bf8dd2250e4a91fa96aac1db2c788e4572b5513d86936efb
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  run_exports:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
```

This is saying that every package that links mvapich must also have a runtime dependency on all compilers.  I assume that's not right? Can I ask what your goal is with the compiler dependencies? Then maybe we can find a way to get what you want.

While we're here, though, it would be good to hae `run_exports` on mvapich itself:

```suggestion
  run_exports:
    - {{ pin_subpackage('mvapich', max_pin='x.y') }}
```

---

OK